addons/account/models/account_move_line.py

```diff
@@ -576,17 +576,21 @@ def _compute_account_id(self):
                 )[0]
         for line in self:
             if not line.account_id and line.display_type not in ('line_section', 'line_note'):
-                line.account_id = line.company_id.account_journal_suspense_account_id
+                previous_two_accounts = line.move_id.line_ids.filtered(
+                    lambda l: l.account_id and l.display_type == line.display_type
```

Comment:
I'm not sure about `l.display_type == line.display_type`. To me, this code should only be called on misc operations and then, display_type is never since you already exclude 'line_section' and 'line_note'

Reply:
If you are here, it means that you didn't find an account from the product, the partner or anything else.
So I think this behavior is good